xbmc/cores/paplayer/VideoPlayerCodec.h

```diff
@@ -41,24 +41,24 @@ class VideoPlayerCodec : public ICodec
 private:
   CAEStreamInfo::DataType GetPassthroughStreamType(AVCodecID codecId, int samplerate, int profile);
 
-  CDVDDemux* m_pDemuxer;
+  CDVDDemux* m_pDemuxer{};
```

Comment:
For raw pointers seems more clear use `{nullptr}`

Also is not the case here but consider this code:

```c++
    int num = 3;
    int* p_num = nullptr;

    p_num = &num;
```
then 

`p_num = {};`

this is same that `p_num = nullptr` and `num` still == 3

but 

`*p_num = {};`  

means `num == 0`  and `p_num` still holds address of `num`

This confusion has happened here: https://github.com/xbmc/xbmc/pull/21679/commits/2172f41d12c67df5f1887a722eac09ade29ac52c

Reply:
Well put. This suggestion is making it into our code guidelines at https://github.com/xbmc/xbmc/pull/22639